params/config.go

```diff
@@ -293,6 +293,7 @@ type ChainConfig struct {
 	PetersburgBlock     *big.Int `json:"petersburgBlock,omitempty"`     // Petersburg switch block (nil = same as Constantinople)
 	IstanbulBlock       *big.Int `json:"istanbulBlock,omitempty"`       // Istanbul switch block (nil = no fork, 0 = already on istanbul)
 	EWASMBlock          *big.Int `json:"ewasmBlock,omitempty"`          // EWASM switch block (nil = no fork, 0 = already activated)
+	EIP2384Block        *big.Int `json:"eip2384Block,omitempty"`        // Eip-2384 (bomb delay) switch block (nil = no fork, 0 = already activated)
```

Comment:
Nitpick, please move this above EWASM, since that's an experimental feature only.